bin/test/main.ml

```diff
@@ -375,12 +376,10 @@ let run_exn ()
   Hashtbl.iter (write_parts ~force_output) files;
   0
 
-let run ()
-    non_deterministic not_verbose syntax silent verbose_findlib prelude
-    prelude_str file section root direction
-  =
+let run setup non_deterministic not_verbose syntax silent verbose_findlib
+    prelude prelude_str file section root direction =
```

Comment:
Here the `force_output` argument is missing, so the call to `run_exn` below is partially applied and exceptions are never catch.
Could you add it and add a type annotation to the function (`int`) to prevent it in the future?

Reply:
Good catch; done.